Thoughtless-Eels/Bangazon-WebApp
# Gw product category

## Link to Ticket
Ticket: # [Link](10)

## Description of Proposed Changes

- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <GW_ProductCategory>

run application
log in as a user
select sell a product
product category should auto-populate with three categories 
```


## Impacted Areas in Application

New file in Data directory: DbInitializer.cs
Program.cs
* 

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
